lib/services/common-api-presenter.js

```diff
@@ -209,28 +211,24 @@ function CommonApiPresenterFactory(
             var scope = self.res.locals.scope;
 
             var promises = [
-                lookupService.ipAddressToMacAddress(self.res.locals.ipAddress),
-                templates.get(name, scope)
+                Promise.props({
+                    server: configuration.get('apiServerAddress', '10.1.1.1'),
+                    port: configuration.get('apiServerPort', 80),
+                    ipaddress: self.res.locals.ipAddress,
+                    netmask: configuration.get('dhcpSubnetMask', '255.255.255.0'),
+                    gateway: configuration.get('dhcpGateway', '10.1.1.1'),
+                    macaddress: lookupService.ipAddressToMacAddress(self.res.locals.ipAddress),
+                    sku: Env.get('config', {}, [ scope[0] ]),
```

Comment:
The array is default to an array in http-service at the start of the middleware and stays an array here: https://github.com/RackHD/on-http/pull/144/files#diff-51cccc923efc7fc45ae2bcf6846680c5R57